plugin/src/main/java/net/pl3x/map/configuration/Advanced.java

```diff
@@ -316,12 +319,12 @@ Setting a color to black (#000000) will make it invisible.""")
         put(Blocks.CRACKED_POLISHED_BLACKSTONE_BRICKS, 0x2C262C);
         put(Blocks.CRACKED_STONE_BRICKS, 0x767676);
         put(Blocks.CRAFTING_TABLE, 0x7B4B2B);
-        put(Blocks.CREEPER_HEAD, 0x74AE6E);
-        put(Blocks.CREEPER_WALL_HEAD, 0x74AE6E);
+        put(Blocks.CREEPER_HEAD, 0x000000);
```

Comment:
These head items should show up on the map as well, imho. Even the player ones (even though those can be textured by owner, we'll just color those brown like steve's hair).